src/site/content/en/blog/pwas-on-oculus-2/index.md

```diff
@@ -0,0 +1,348 @@
+dynamic workspace that empowers teams to manage projects, automate workflows, and rapidly build new
+solutions.
+
+More PWAs like Slack, Dropbox, or Canva are coming, as teased in a
+[video](https://www.facebook.com/watch/?v=4637130066326723) featuring Jacob Rossi that was released
+in the context of Facebook's Connect conference in 2021.
+
+## Creating PWAs for Oculus
+
+Meta has outlined the required steps in their
+[documentation](https://developer.oculus.com/documentation/web/pwa-gs/). In general, PWAs that are
+installable in Chrome should oftentimes work out of the box.
+
+### Web App Manifest requirements
+
+There are some important differences compared to
+[Chrome's installability criteria](https://web.dev/install-criteria/#criteria) and the
+[Web App Manifest spec](https://w3c.github.io/manifest/), for example regarding language constraints
+(Oculus only supports left-to-right languages at the moment) or the optionality of `start_url`,
+which Chrome strictly requires for an app to be installable. Oculus offers a
+[command line tool](https://developer.oculus.com/documentation/web/pwa-packaging/) that developers
+can use to create PWAs for the Oculus Quest&nbsp;2, which allows them to pass the missing (or
+override the existing) parameters in the Web App Manifest.
+
+| Name         | Required | Description                                                                                                        |
+| ------------ | -------- | ------------------------------------------------------------------------------------------------------------------ |
+| `name`       | yes      | The name of the PWA. Currently Oculus only supports left-to-right languages for the name.                          |
+| `display`    | yes      | Currently this value must be `"standalone"` or `"minimal-ui"`. Oculus currently does not support any other values. |
+| `short_name` | no       | A shorter version of the app name, if needed.                                                                      |
+| `scope`      | no       | This field allows developers to specify what URL or paths should be considered as part of the app.                 |
+| `start_url`  | no       | Developers can specify a starting URL to load.                                                                     |
+
+Oculus has a number of optional **proprietary** Web App Manifest fields that can be used to
+customize the PWA experience.
+
+| Name                    | Required | Description                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
The proposal for the previous table would let you get rid of the 'Required' column. Here it works because you've already established that only required items are labeled and you've stated in the introduction that these are all optional. To emphasize the point, you could move the word 'optional' inside the bold markdown.

Reply:
Done.